packages/shared/src/rpc/index.ts

```diff
@@ -115,6 +115,8 @@ export const getSwappingEnvironment = createRequest(
 
 const ingressEgressEnvironment = z.object({
   minimum_deposit_amounts: chainAssetNumberMap,
+  ingress_fees: chainAssetNumberMap.optional(),
+  egress_fees: chainAssetNumberMap.optional(),
```

Comment:
We simply won't deploy this to an env w/o the new fields

```suggestion
  ingress_fees: chainAssetNumberMap,
  egress_fees: chainAssetNumberMap,
```